spec/lib/guardian/flag_guardian_spec.rb

```diff
@@ -13,10 +13,12 @@
       expect(Guardian.new(admin).can_create_flag?).to eq(true)
       expect(Guardian.new(user).can_create_flag?).to eq(false)
 
-      Fabricate(:flag)
+      flag = Fabricate(:flag)
 
       expect(Guardian.new(admin).can_create_flag?).to eq(false)
       expect(Guardian.new(user).can_create_flag?).to eq(false)
+
+      flag.destroy!
```

Comment:
Maybe we should do this before all tests? Delete all custom flags?

https://github.com/discourse/discourse/blob/78e9922a84ff2a883e3b1a74e46db2c395bfd40b/spec/rails_helper.rb#L110-L112

Reply:
I just checked and turbo rspec is running around the same time so there would be no performance problem with deleting all custom flags after each test. 

On the other hand, flags are created only in a few specs, so I think that I would prefer to delete them explicitly. Hopefully, now all places are found :) 

Offtopic: I just checked and we cannot do it before each test because when you have `fab!` on top, that flag is deleted in the before block.